math/exprtk/Portfile

```diff
@@ -0,0 +1,25 @@
+PortSystem                  1.0
+PortGroup                   github 1.0
+
+github.setup                ArashPartow exprtk 0.0.3
+github.tarball_from         archive
+categories                  math devel
+maintainers                 @arashpartow
```

Comment:
this ideally should include your obfuscated e-mail address as I said before. Additionally, please add "openmaintainer" unless there is a strong reason not to do so; that will allow others to make updates/fixes to the port without explicit approval.

Reply:
I've set this to `openmaintainer`, so that anyone can update the `Portfile `as needed.